app/src/main/res/layout/download_dialog.xml

```diff
@@ -106,4 +107,16 @@
             android:max="31"
             android:progress="3" />
     </LinearLayout>
+
+    <org.schabi.newpipe.views.NewPipeTextView
+        android:id="@+id/streams_hidden"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/threads_layout"
+        android:layout_marginLeft="24dp"
+        android:layout_marginRight="24dp"
+        android:layout_marginBottom="6dp"
+        android:textAlignment="textEnd"
```

Comment:
Uhm, I would center this, it would look nicer. Also made other small changes. (already done)

![image](https://user-images.githubusercontent.com/36421898/174449259-1d30874e-cf98-46bd-94b8-18c7bce88bfa.png)